src/go/rpk/pkg/cli/cluster/partitions/list.go

```diff
@@ -1,197 +1,225 @@
+// Copyright 2023 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
 package partitions
 
 import (
+	"context"
+	"errors"
 	"fmt"
+	"net/http"
 	"strconv"
 	"strings"
+	"sync"
 
-	"github.com/docker/go-units"
 	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/adminapi"
 	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/config"
 	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/out"
 	"github.com/spf13/afero"
 	"github.com/spf13/cobra"
 	"github.com/twmb/types"
+	"go.uber.org/zap"
+	"golang.org/x/sync/errgroup"
 )
 
-func newListPartitionMovementsCommand(fs afero.Fs, p *config.Params) *cobra.Command {
+func newListCommand(fs afero.Fs, p *config.Params) *cobra.Command {
 	var (
-		completion       int
-		all              bool
-		human            bool
-		partitions       []string
-		response         []adminapi.ReconfigurationsResponse
-		filteredResponse []adminapi.ReconfigurationsResponse
+		all             bool
+		disabledOnly    bool
+		partitions      []int
+		logFallbackOnce sync.Once
 	)
 	cmd := &cobra.Command{
-		Use:   "move-status",
-		Short: "Show ongoing partition movements",
-		Long:  helpListMovement,
-		Run: func(cmd *cobra.Command, topics []string) {
-			p, err := p.LoadVirtualProfile(fs)
-			out.MaybeDie(err, "unable to load config: %v", err)
-			out.CheckExitCloudAdmin(p)
+		Use:     "list [TOPICS...]",
+		Aliases: []string{"ls", "describe"},
+		Short:   "List partitions in the cluster",
+		Long: `List partitions in the cluster
 
-			// If partition(s) is specified but no topic(s) is specified, exit.
-			if len(topics) <= 0 && len(partitions) > 0 {
-				out.Die("specify at least one topic when --partition is used, exiting.")
-			}
+This commands lists the cluster-level metadata of all partitions in the cluster.
```

Comment:
typo: commands -> command

Reply:
Thanks! Fixed in e8c2beabb71490112fd68887de7f0c8d4bd57811